FirebaseABTesting/Tests/Unit/FIRExperimentControllerTest.m

```diff
@@ -186,9 +191,13 @@ - (void)testUpdateExperiments {
                                                    policy:
                                                        ABTExperimentPayload_ExperimentOverflowPolicy_DiscardOldest  // NOLINT
                                             lastStartTime:now
-                                                 payloads:payloads];
+                                                 payloads:payloads
+                                        completionHandler:^(NSError *_Nullable error) {
+                                          completionHandlerCalled = YES;
+                                        }];
 
   XCTAssertEqual([_mockCUPController experimentsWithOrigin:gABTTestOrigin].count, 2);
+  XCTAssertTrue(completionHandlerCalled);
```

Comment:
This seems flaky if the method above is async.

Reply:
I renamed this method, the method itself is not actually async... it's just called from a background thread. Name was confusing.